packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,555 @@
+| focusMode  | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`         | Sets the focus behavior for the Breadcrumb. |
+| size       | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb              |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button - BreadcrumbButton component is used inside BreadcrumbItem.
+- Link - BreadcrumbLink is used inside BreadcrumbItem.
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbButton onClick={() => console.log('smth...')}>
+    Item 2
+  </BreadcrumbButton>
+</BreadcrumbItem>
+<BreadcrumbItem>
+  <BreadcrumbLink href="#">
+    Item 3
+  <BreadcrumbLink>
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default  | Purpose                                     |
+| ------------ | --------------------------------------- | -------- | ------------------------------------------- |
+| active       | boolean                                 | false    | Indicates if the item is active             |
+| disabled     | boolean                                 | false    | Disables Breadcrumb item                    |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`    | Sets the focus behavior for the Breadcrumb. |
+| icon         | _slot_                                  |          | Sets icon                                   |
+| iconPosition | `before`, `after`                       | `before` | Sets icon position                          |
+
+#### Breadcrumb icon
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbButton icon={<IconComponent />} iconPosition="after">Item 1</BreadcrumbButton>
+<BreadcrumbLink icon={<IconComponent />}>Item 1</BreadcrumbLink>
+```
+
+### BreadcrumbDivider
+
+#### DOM
+
+```HTML
+  <li class="fui-BreadcrumbDivider">
+    <svg
+      aria-hidden="true"
+      fill="currentColor"
+      height="16"
+      viewBox="0 0 16 16"
+      width="16"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path d="M5.65 3.15a.5.5 0 000 .7L9.79 8l-4.14 4.15a.5.5 0 00.7.7l4.5-4.5a.5.5 0 000-.7l-4.5-4.5a.5.5 0 00-.7 0z" fill="currentColor" />
+    </svg>
+  </li>
+```
+
+#### API
+
+| Property | Values             | Default   | Purpose              |
+| -------- | ------------------ | --------- | -------------------- |
+| variant  | `chevron`, `slash` | `chevron` | Sets type of divider |
```

Comment:
Not sure that we want to have this as it creates a strange API:

```ts
<BreadcrumbDivider variant="slash">
  <FooIcon />
</BreadcrumbDivider>
```

What will be rendered? 🤔

---

I was thinking about following:

```ts
function App() {
  return (
    <>
      <BreadcrumbDivider /> {/* renders chevron */}
      <BreadcrumbDivider>/</BreadcrumbDivider> {/* renders a slash, from the design spec it's unclear if it's an icon */}
      <BreadcrumbDivider><FooIcon /></BreadcrumbDivider>
    </>
  )
}
```

Reply:
dividerType is passed from the Breadcrumb component